source/includes/php-frameworks/Restaurant.php

```diff
@@ -0,0 +1,32 @@
+<?php
```

Comment:
`includes/php-frameworks` seems like a generic directory that might end up with more content unrelated to this PR. I'd suggest grouping all of the symfony-related files under a "symfony" sub-directory.